manifests/params.pp

```diff
@@ -52,4 +52,9 @@
   # Tool used by this module to fetch JBoss installation files from network
   $fetch_tool       = hiera('jboss::params::fetch_tool', 'jboss::internal::util::download')
 
+  # Additional commandline parameters passed to java
+  $server_opts      = hiera('jboss::params::server_opts', undef)
```

Comment:
Options should have sensible, working defaults. Best to use values already used by JBoss